tests/unit/tabular/test_base.py

```diff
@@ -224,6 +350,130 @@ def test__sample_rows_previous_rows_appended_correctly():
     pd.testing.assert_frame_equal(sampled, expected)
 
 
+def test_sample_empty_transformed_conditions():
```

Comment:
Can we just rename any tests that use the new methods to have that method name in them?